src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/directionissued/DirectionIssuedAboutToStartHandler.java

```diff
@@ -4,6 +4,7 @@
 import static uk.gov.hmcts.reform.sscs.ccd.domain.DirectionType.*;
 import static uk.gov.hmcts.reform.sscs.ccd.presubmit.directionissued.ExtensionNextEventItemList.*;
 import static uk.gov.hmcts.reform.sscs.helper.SscsHelper.getPreValidStates;
+import static uk.gov.hmcts.reform.sscs.util.OtherPartyDataUtil.isOtherPartyPresent;
```

Comment:
Typically helper methods like this one should not be statically imported. As a rule of thumb, if it is a commonly used and understood method (eg assertThat from Assertions, or isYes from our own code) then it is fine. However, this method is quite niche, so it's recommended to keep it as OtherPartyDataUtil.isOtherPartyPresent in the code to show where it is coming from. Otherwise it could be confused as a local method, esp in a large file.